src/devhub/devhub.js

```diff
@@ -56,8 +56,9 @@ async function main_seeds() {
     "https://api.github.com/repos/tigerbeetle/tigerbeetle/issues?per_page=200";
 
   const [records, issues] = await Promise.all([
-    (async () => await (await fetch(data_url)).json())(),
-    (async () => await (await fetch(issues_url)).json())(),
+    (async () => await (await fetch(data_url, { cache: "no-cache" })).json())(),
+    (async () =>
+      await (await fetch(issues_url, { cache: "no-cache" })).json())(),
```

Comment:
Let's introduce a `fetch_json` helper here! This pattern of `(async => await await)()` looks less fun now than when it was originally written 🤣 

I _think_ we should be able to do just

```
await Promise.all([
    fetch_json(data_url),
    fetch_json(issues_url), 
])
```

which looks much tamer.